taxi/forms.py

```diff
@@ -14,7 +14,7 @@ class CarForm(forms.ModelForm):
 
     class Meta:
         model = Car
-        fields = "__all__"
+        fields = ("models", "manufacturer", "drivers")
```

Comment:
The field name 'models' in the `fields` tuple should likely be 'model' to match the expected field in the `Car` model. Please verify the field names in the `Car` model to ensure they match.